src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -802,6 +802,46 @@ public String toString() {
     }
   };
 
+  public static final Builder<Map.Entry<StreamEntryID, List<StreamEntry>>> STREAM_AUTO_CLAIM
```

Comment:
Add a recognizable suffix with the variable name.

Reply:
Hi @sazzad16 Can you please clarify here ? I didn't get what you suggested here.
`STREAM_AUTO_CLAIM, STREAM_AUTO_CLAIM_ID` are the builders for xautoclaim & xautoclaimJustId commands respectively.

Reply:
@s-sathish Almost all the builders here in BuilderFactory represent an (dedicated) object/collection of (dedicated) object. In other cases, we should make a separation. Only thing here we can separate is the variable name. This what I'm asking here.

I.e. you can use `STREAM_AUTO_CLAIM_RESPONSE` and `STREAM_AUTO_CLAIM_ID_RESPONSE` as names.

Reply:
@s-sathish Almost all the builders here in BuilderFactory represent an (dedicated) object/collection of (dedicated) object. In other cases, we should make a separation. Only thing here we can separate is the variable name. This is what I'm asking here.

I.e. you can use `STREAM_AUTO_CLAIM_RESPONSE` and `STREAM_AUTO_CLAIM_ID_RESPONSE` as names.

Reply:
Got it. Changes done.